### PR TITLE
Release 2.31.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,57 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.31.0](https://github.com/serverless/serverless/compare/v2.30.3...v2.31.0) (2021-03-23)
```
Comment (pgrzesik) [inline CHANGELOG.md:5]: I was not 100% sure about keeping all of the entries in Bug Fixes & Maintenance Improvements - please let me know if something else should be removed as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,57 @@
...
+
+### Bug Fixes
+
+- **Packaging:** Ensure to properly exclude dependencies when NODE_ENV set ([#9142](https://github.com/serverless/serverless/pull/9142)) ([6e3e21c](https://github.com/serverless/serverless/commit/6e3e21cafaa053a0f30239513eac5c7a725b7694)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:**
+  - Recognize "--stage" as provider agnostic option ([#9151](https://github.com/serverless/serverless/pull/9151)) ([271ac82](https://github.com/serverless/serverless/commit/271ac8281c10e23ce609165582b3b996b60d5bd1)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Do not show options order deprecation info with help request ([#9151](https://github.com/serverless/serverless/pull/9151)) ([ad8f9b0](https://github.com/serverless/serverless/commit/ad8f9b059740bb2581445457b6751e918a57e585)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Ensure all expected command schemas in collections ([#9168](https://github.com/serverless/serverless/pull/9168)) ([031ca4d](https://github.com/serverless/serverless/commit/031ca4dd2aa35b3a15e00db0601a11b7b25d3b4d)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:20]: Let's remove that one (it was fix against `master` and not against previous release)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,57 @@
...
+  - Fix reference to AWS service options ([#9168](https://github.com/serverless/serverless/pull/9168)) ([6a8ea90](https://github.com/serverless/serverless/commit/6a8ea9051865ea4975e335d952de3695d044a0f3)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:21]: Same here

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,57 @@
...
+  - Expose resolved string command by `resolveInput` util ([#9151](https://github.com/serverless/serverless/pull/9151)) ([362f5e9](https://github.com/serverless/serverless/commit/362f5e94e044f51fe6e9b2f9de4de810fb9f8d04)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Improve flow documentation ([#9140](https://github.com/serverless/serverless/pull/9140)) ([851c9f4](https://github.com/serverless/serverless/commit/851c9f4d52a69c3ac0f15f5fb353ae9f7a554530)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Improve setup of resolution flow ([#9140](https://github.com/serverless/serverless/pull/9140)) ([ab1c673](https://github.com/serverless/serverless/commit/ab1c673bb2f7d4279d0406f355516d616e1f00c8)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Integrate isHelpRequest into resolveInput util ([#9140](https://github.com/serverless/serverless/pull/9140)) ([a481170](https://github.com/serverless/serverless/commit/a48117041c3993f5ba969329dd605f6a80efcbad)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:41]: It might be nice to style all programmatic names with back ticks

**Review by medikoo — COMMENTED** [review]
@pgrzesik looks good, just few style suggestions

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**